tests/4.5/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_firstprivate.c

```diff
@@ -36,17 +40,37 @@ int test_target_teams_distribute_parallel_for_firstprivate() {
   }
 
   // check multiple sizes. 
-#pragma omp target data map(to: a[0:SIZE_N], b[0:SIZE_N], c[0:SIZE_N]) map(from: d[0:SIZE_N])
+#pragma omp target data map(to: a[0:SIZE_N], b[0:SIZE_N], c[0:SIZE_N])
   {
-#pragma omp target teams distribute parallel for firstprivate(privatized)
+#pragma omp target teams distribute parallel for firstprivate(privatized, firstized) num_teams(10) num_threads(256)
```

Comment:
Thanks for addressing this. Although I didnt test if it works now for us, I have two (minor) concerns:
(1) `privatized` should be `private` now instead of `firstprivate`, because its initial value is not used at all, is it?
(2) For "our" device (SX-Aurora) hard-coded `num_threads(256)` might be a problem, because we only have eight cores. Since the device is not running an OS, the context switch is done by involving the host, which is quite expensive.

Reply:
I wanted to keep the privatized in the `firstprivate` clause as we are trying to test if this clause actually does the privatization. But for an application you are correct. 

This threads number is clearly an issue. Uhmm I wonder if we should try to expose these configurations to the outside world. IT would require us to change a lot of tests though. Interesting point to consider. 

My I ask what system are you running on? It's ok if you cannot say.

Reply:
Ok, I see. Then it makes sense to keep it in the `firstprivate` clause.

I think the `num_threads` configuration is not generic enough. However, I am fine with your suggestion in issue #13, of course.